src/landing_page.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>OPTIMADE Index Meta-databse</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="keywords" content="optimade,materials,crystals,database">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="shortcut icon" type="image/x-icon" href="https://www.optimade.org/optimade-python-tools/images/favicon.png">
+    <style>
+        html { margin: 2em; }
+        h4, p { margin-left: 2em; }
+        span.version {background-color: lightgrey; font-weight: bold; font-family: monospace; padding: 0em 0.5em 0em 0.5em; border-radius: 1em;}
+    </style>
+    </head>
+    <body>
+        <img id="logo" width="100" src="https://avatars0.githubusercontent.com/u/23107754?s=400&u=e1580af496ac195a3ac4c445a66a5699efb0c3d3"></img>
+        <h3>This is the base URL of an <a href="https://www.optimade.org">OPTIMADE</a> index meta-database, served statically from <i class="fa fa-github"></i> <a href="https://github.com/Materials-Consortia/providers">Materials-Consortia/providers</a> on GitHub.</h3>
+        <h4>Two endpoints can be appended to the current URL: </h4>
+        <ul>
+            <li><span class="version">/v1/info/</span>, which serves information about this provider.</li>
+            <li><span class="version">/v1/links/</span>, which serves links to the databases served by this provider.</li>
+        </ul>
```

Comment:
```suggestion
        </ul>
        <p>Note that if the /v1 part should only be added once to the URL path.</p>
```
We should add something like this for completeness.

Reply:
Please change the HTML tag appropriately.

Reply:
Does fee000b3417b5aafc4d64faa8d59f711ee35f1cf resolve this?